src/mixins/reqFulfillment.js

```diff
@@ -51,13 +51,49 @@ export default {
       reqString = reqString.replace(/''/g, '"').replace(/,[\s]+/g, ',');
       const splitReq = reqString.split(/(,|\(|\)|\/)/);
       const _this = this;
+      let skipNumber;
       for (let i = 0; i < splitReq.length; i++) {
-        if (splitReq[i].indexOf('"') >= 0) {
-          // Set strings (like "permission of instructor") automatically to false
+        if (splitReq[i].indexOf('"') >= 0 && i !== skipNumber) {
+          // if the requirement is a string instead of a class ID:
+          // If the string is "One subject in X", check for any subject with X in their ID
+          // things that are still not handled correctly:
+          // 21M.283 doesn't work correctly because film / music
+          // 24.280, 21L.709, 21L.715, 21L.S96, 21L.S97 don't work because we don't keep the count between i's
+          // Set other strings (like "permission of instructor") automatically to false
           // If required as an alternative to other prereqs, will not affect fulfillment
           // If the string is a required prereq, it must be manually dismissed
-          splitReq[i] = 'false';
-        } else if ('()/, '.indexOf(splitReq[i]) < 0) {
+          const req = splitReq[i];
+          let idCategory;
+          let numRequired;
+          // the 'ne subject' is on purpose because some say 'One' and some say 'one'
+          if (req.indexOf('ne subject in') >= 0 || req.indexOf('wo subjects in') >= 0) {
+            if (req.indexOf('ne subject in') >= 0) {
+              numRequired = 1;
```

Comment:
Are there any cases where it's some other number of subjects?  There are [libraries to convert words to numbers](https://github.com/finnfiddle/words-to-numbers) if we want to make this more extendable, although it's always good to keep our node_modules down.